### ISSUE TITLE
Cells randomly disappearing and titleLabel.text displays wrong text

### ISSUE THREAD

**Issue by Marolicious** [younatics/ExpandableCell#23]
Hey. I have 11 ExpandableCells and about 50 ExpandedCells. I am initializing it like that:

`func expandableTableView(_ expandableTableView: ExpandableTableView, expandedCellsForRowAt indexPath: IndexPath) -> [UITableViewCell]? {`

		`var cells: [ExpandedCell] = []
		for i in 0...platinumProteinMeals[indexPath.row].meals.count - 1 {
			let cell = tableView.dequeueReusableCell(withIdentifier: ExpandedCell.ID) as! ExpandedCell
			print("\(indexPath) | \(i) | \(platinumProteinMeals[indexPath.row].meals[i])")
			cell.titleLabel.text = platinumProteinMeals[indexPath.row].meals[i]
			cell.isHidden = false
			cells.append(cell)
		}
		return cells
    }`

The platinumProteinMeals is an array of 11 NDDSections, one section would look like this:
`NDDSection(category: "Beef", meals: ["Beef Avocado Salad", "Beef in Chili Sauce", "Beef Soup", "Beef Steak", "Beef Tartar", "Beef with French Beans", "Chilli Lime Beef", "Thai Beef Salad"])`

My issue is that the cell randomly disappears (mostly while scrolling) and a wrong `meal` of the`meals` array is getting displayed. For example a Turkey meal is being displayed in the Beef ExpandableCell. Any idea how to fix this? Thanks in advance

**Comment by andrei0929** [comment]
I have the same issue. When scrolling, the expanded cells randomly disappear
![image uploaded from ios](https://user-images.githubusercontent.com/10848005/32768563-fce41158-c920-11e7-9a65-5f99b927620c.png)

**Comment by NrmeenTomoum** [comment]
@younatics  I have same problem

**Comment by NrmeenTomoum** [comment]
@andrei0929  @Marolicious  @kwongius  @ezefranca  @kennydust  
I added
 tableView.reloadData() 
in the end of the the function of 
 public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) 
waiting you respond if this helped you or not

**Comment by NrmeenTomoum** [comment]
@Marolicious  @andrei0929  @kwongius @ezefranca @kennydust  waiting your respond

**Comment by andrei0929** [comment]
Although I wasn't implementing the didSelectRowAt indexPath method, I called reloadData in the method but it didn't help me

**Comment by NrmeenTomoum** [comment]
are you added  tableView.reloadData() in the End of  didSelectRowAt?

**Comment by damir18yo** [comment]
@NrmeenTomoum @andrei0929 I have same problem. Did you find any solution other than `tableView.reloadData()`?

**Comment by NrmeenTomoum** [comment]
@damir18yo  I didn't do any think more than added  tableView.reloadData() at the End of didSelectRowAt func in ExpandableTableView.swift File created by @younatics after that disappeared cell problem appears from scrolling disappeared .

**Comment by damir18yo** [comment]
@NrmeenTomoum I tried your solution but animation becomes very weird

**Comment by NrmeenTomoum** [comment]
try to made animation of table . middle @damir18yo

**Comment by akominch** [comment]
Faced the same problem. Maybe someone knows a good solution to this problem?

**Comment by Sunnymous** [comment]
Facing the same problem. 
I'll try to contribute if I find out the solution to this, but is this project deserted?

**Comment by FaridHaddad** [comment]
This is happening to me too. Did anyone solve this problem ?

**Comment by Sunnymous** [comment]
@FaridHaddad Nope. I`ll try to, but I don`t think I can. Finding a new library would be better.

**Comment by superhard** [comment]
Which library can be a good alternative?

**Comment by Sunnymous** [comment]
@superhard Well, this still is a ExpandableCell's thread, so I can't discuss other libraries here.

**Comment by jonasrotilli** [comment]
Multiple reports and no definitive solution to the problem?

**Comment by jonasrotilli** [comment]
In addition, in the example application itself of this git the problem happens: expands all and goes opening and closing, and fiddling with scrool.
You will see that they add up other unrelated items.

**Comment by AhmedAmineZribi** [comment]
any one have found solution for this issue ??

**Comment by React-360quinlan** [comment]
@akominch https://ezio-difficulty48.github.io/8CpuQDqp3JLwZVYs4SSzbplRBI/
